lib/methods/createMany.js

```diff
@@ -0,0 +1,30 @@
+var async = require('async');
+
+/**
+ * Creates multiple models.
+ *
+ * Utilizes batch operation if available or falls back to the default connector
+ * implementation and manually creates models one by one.
+ *
+ * @param {Arrow.Model} Model The model class being updated.
+ * @param {Array<Object>} values Array of attributes to set on the new models.
+ * @param {Function} callback Callback passed an Error object (or null if successful), and info about the batch operation.
+ * @throws {Error}
+ */
+exports.createMany = function (Model, values, callback) {
+	// Use batch delete if possible
+	var currentUser = this.user || this.baseUser;
+	var isAdminUser = currentUser.admin === 'true';
+	var modelSupportsBatchCreate = typeof Model.createMany === 'function';
```

Comment:
Sorry, my bad. I missed the changes that are required to the arrowdb node sdk. You need to test this together with https://github.com/appcelerator/arrowdb-node-sdk/pull/12